app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<% @recipes.each_with_index do |recipe,index|  %>
+<%= link_to recipe_path(recipe.id) do %>
+<div class='recipe-container'>
+<section class='recipe-info'>
+   <h3 class='recipe-1'>Recipe <%= index+1 %></h3>
+   <a class='recipe-btn'><%= button_to "Remove", recipe_path(recipe), method: :delete, data: { turbolinks: false } %></a>
+</section>   
+   <p class='recipe-description'><%= recipe.description %></p>
+</div>  
+ <% end %>  
+<% end %>
+
```

Comment:
- [ ] [MANDATORY] Just as the previous reviewer mentioned [here](https://github.com/ismayito/Rails-recipe-app/pull/29#discussion_r1459885540), when I navigate to the recipes page, I see nothing.

![image](https://github.com/ismayito/Rails-recipe-app/assets/86318284/4b274a53-1219-49a1-ba58-156bd6171ce5)